schema/deploy/computed_columns/form_change_gross_payments_to_date.sql

```diff
@@ -0,0 +1,21 @@
+-- Deploy cif:computed_columns/form_change_gross_payments_to_date to pg
+-- requires: tables/form_change
+
+begin;
+
+create or replace function cif.form_change_gross_payments_to_date(parameter_fc cif.form_change)
+returns numeric as
+$fn$
+
+  select
+    round(sum(coalesce((fc.new_form_data->>'adjustedGrossAmount')::numeric, (fc.new_form_data->>'calculatedGrossAmount')::numeric, (fc.new_form_data->>'maximumAmount')::numeric)), 2)
```

Comment:
I wonder if in the `payments_to_date` we should include the maximum amounts, which are payments that haven't been made yet

Reply:
This is what I understood from talking with @nanyangpro but we can wait for confirmation before merging

Reply:
You're right. We are not to include the maximum amounts here, only completed milestones (ones with adjusted/calculated values) are to be counted. Fixed.